# As we have a git version without the git metadata, we tell it which commit we have used
# by wpoely86@gmail.com
diff -ur psi4public-7baee9b8ff6e30b39a7ee627750a518c39bd84f3.orig/src/bin/psi4/gitversion.py psi4public-7baee9b8ff6e30b39a7ee627750a518c39bd84f3/src/bin/psi4/gitversion.py
--- psi4public-7baee9b8ff6e30b39a7ee627750a518c39bd84f3.orig/src/bin/psi4/gitversion.py	2015-08-14 04:47:54.000000000 +0200
+++ psi4public-7baee9b8ff6e30b39a7ee627750a518c39bd84f3/src/bin/psi4/gitversion.py	2015-08-14 12:50:26.702794481 +0200
@@ -80,4 +80,5 @@
     ghash = ''
     mmp = ''
 
-write_version(branch, mmp, ghash, status)
+#write_version(branch, mmp, ghash, status)
+write_version("master", None, "7baee9b8ff6e30b", "")
